src/script/components/MessagesList/Message/ContentMessage/asset/index.tsx

```diff
@@ -64,14 +65,17 @@ const ContentAsset = ({
         <>
           {(asset as Text).should_render_text() && (
             <div
+              role="button"
+              tabIndex={0}
               className={`text ${includesOnlyEmojis((asset as Text).text) ? 'text-large' : ''} ${
                 status === StatusType.SENDING ? 'text-foreground' : ''
               } ${isObfuscated ? 'ephemeral-message-obfuscated' : ''}`}
               dangerouslySetInnerHTML={{__html: (asset as Text).render(selfId, message.accent_color())}}
               onClick={event => onClickMessage(asset as Text, event)}
+              onKeyDown={event => handleKeyDown(event, onClickMessage.bind(this, asset as Text, event))}
```

Comment:
```suggestion
              onKeyDown={event => handleKeyDown(event, onClickMessage.bind(null, asset as Text, event))}
```
Doesn't need the lexical `this` here